src/DateInput/DateInput.jsx

```diff
@@ -0,0 +1,123 @@
+// Global Imports
+import PropTypes from 'prop-types';
+
+// Local imports
+import { DEFAULT_CLASS as DEFAULT_LABEL_CLASS } from '../Label/Label';
+import { classBuilder } from '../utils/Utils';
+import './DateInput.scss';
+import TextInput from '../TextInput/TextInput';
```

Comment:
```suggestion
import TextInput from '../TextInput';
```

There's a barrel file (`index.js`) sitting inside of each component directory so you don't need the second `TextInput` here.

Reply:
Updated to use the barrel file